src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/BuildPropertyPage.VisualBasic.xaml.cs.xlf

```diff
@@ -189,8 +189,8 @@
       </trans-unit>
       <trans-unit id="EnumProperty|DebugType|DisplayName">
         <source>Debug symbols</source>
-        <target state="translated">Symboly ladění</target>
-        <note />
+        <target state="needs-review-translation">Symboly ladění</target>
+        <note>="PDB Only" /&gt;</note>

```

Comment:
This happened on the files in this PR. Is this something broken in that particular property or was it a change in the XliffTasks?

Reply:
It wasn't a change in XliffTasks, so pretty sure something needs updating on that particular property

Reply:
There wouldn't be a change here unless either:
- You changed this property in the PR, or
- The XliffTasks changed its understanding of what was already there

Since I don't see a change for this property, it seems like maybe there is now `note` support for XAML. I'm not certain without checking if this existed prior. We use it in RESX files. Looking very quickly, you can see it is partially reading this comment now: https://github.com/dotnet/project-system/blob/04bae84071dd08f6e5f5c3e9c9f39a2d349f7654/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.VisualBasic.xaml#L380-L401

So, functionality did change from the prior XliffTasks package. I don't know what the conditions are for an XAML comment to be considered as a `note`. FYI, these notes are used by the translation team, so we shouldn't send them garbage notes.

Reply:
Haha, I just noticed this too (in #8809) and filed a bug to track it: https://github.com/dotnet/xliff-tasks/issues/719